core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.ts

```diff
@@ -301,6 +389,21 @@ export class TranslationSuggestionReviewModalComponent implements OnInit {
     this.init();
   }
 
+  showNextItemToReview(suggestionId: string): void {
```

Comment:
How does this differ from `gotoNextItem`?

Reply:
This function is called when the user reviews the translation ('accept' or 'reject'). This function adds the current suggestionId to the resolvedSuggestionIds array (marking it as reviewed) and then calls gotoNextItem. The gotoNextItem function doesn't mark anything as resolved (it isn't called directly by the accept or reject functions)

Reply:
Hmm, can you try to come up with a better name? Smth like `processAndGotoNextItem`

Reply:
Done